lib/classes/PluginManager.js

```diff
@@ -429,6 +430,28 @@ class PluginManager {
     this.getCommand(commandsArray);
   }
 
+  /**
+   * If the command has no use when operated in a working directory with no serverless
+   * configuration file, throw an error
+   */
+  validateServerlessConfigDependency(command) {
+    if ('configDependent' in command && command.configDependent) {
+      const servicePath = this.serverless.config.servicePath || 'x';
```

Comment:
Consider leaving defaulting to `getServerlessConfigFile` now?  (See https://github.com/serverless/serverless/pull/5314/files#diff-045b1def1b55ed8f81130ccff02544a5R10)